eval/src/vespa/eval/eval/value_codec.h

```diff
@@ -5,18 +5,23 @@
 #include "value.h"
 #include "tensor_spec.h"
 #include <vespa/vespalib/stllike/string.h>
+#include <vespa/vespalib/util/exception.h>
 
 namespace vespalib { class nbostream; }
 
 namespace vespalib::eval {
 
+VESPA_DEFINE_EXCEPTION(DecodeValueException, Exception);
```

Comment:
consider moving this to `vespalib/src/vespa/vespalib/util/exceptions.h` and vespalib namespace

Reply:
Then maybe we should combine it with the DeserializationException from document as well?